compute-vars/large.tfvars

```diff
@@ -0,0 +1,7 @@
+## Large Configuration Template
+
+bastion                     = {memory      = "32",   processors  = "2",  "count"   = 1}
+bootstrap                   = {memory      = "64",   processors  = "1",  "count"   = 1}
+master                      = {memory      = "64",   processors  = "1",  "count"   = 3}
+worker                      = {memory      = "64",   processors  = "1",  "count"   = 2}
+
```

Comment:
we can make some modifications to the large
1. processors = "1.5" for all types of nodes
2. Memory = 64 GB for all OCP nodes
3. Worker count = 3

Reply:
Updated the large template.